middlewares/isUser.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-trailing-spaces */
```

Comment:
Why did you disable this rule?

Reply:
I did that while debugging and I forgot to enable it back. But I've taken care of it now